lang/Makefile

```diff
@@ -4,7 +4,10 @@
 # ocaml-tree-sitter must have been built and installed for this to work.
 #
 
-SUPPORTED_LANGUAGES = ruby
+SUPPORTED_LANGUAGES = \
```

Comment:
woohoo! ok after that we need Go and Python, just to avoid having to fix parsing bugs in pfff for those languages,
especially Go.